### PR TITLE
Removes unused packages as reported by depcheck

### PR DESCRIPTION
**Pull request by ryanhofdotgov**
Evaluted unused dependencies reported by `depcheck`; filtered out
those that had references in `webpack.config.js`, `gulpfile.js`,
`eqip.scss`, and `package.json` blocks (excluding `dependencies`).
and `gulpfile.js`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,42 +17,33 @@
     "@18f/stylelint-rules": "^2.0.0",
     "axios": "^0.18.0",
     "axios-mock-adapter": "^1.15.0",
-    "babel-core": "^6.26.3",
-    "babel-jest": "^23.0.1",
```
Comment (afeld) [inline package.json]: Definitely need these two!

Comment (ryanhofdotgov) [reply package.json]: Yarn figures out that we need it – do we need to put it as a top-level dependency?

Comment (ryanhofdotgov) [reply package.json]: (i.e., `jest` pulls it in, so does  `babel-loader`)

Comment (afeld) [reply package.json]:
```
➜  e-QIP-prototype git:(remove-unused) yarn
yarn install v1.7.0
[1/5] 🔍  Resolving packages...
[2/5] 🚚  Fetching packages...
[3/5] 🔗  Linking dependencies...
warning " > babel-loader@7.1.4" has unmet peer dependency "babel-core@6".
...
```

Comment (afeld) [reply package.json]: Good to go after that's re-added!

**Comment by ryanhofdotgov** [comment]
Thanks! I had somehow missed that warning. After reading https://github.com/yarnpkg/yarn/issues/5347 I've realized yarn's dependency management was different than what I was expecting.

Not touching the spec tests since they aren't working yet. Did this clean up  to address some of the "high" severity items in the Synk report on #563

**Comment by afeld** [comment]
> yarn's dependency management was different than what I was expecting

It's A Feature 😉

**Pull request merged**